src/inglued.cpp

```diff
@@ -143,7 +145,7 @@ namespace inglued {
   inline void hikeup_deep_deps(map_deps_t& deps) {
 
     auto scan_inglued_dep = [](const fs::path& glue_file) {
-      auto deep_deps = read_deps(glue_file.native());
+      auto deep_deps = read_deps((const std::string &)glue_file.native());
```

Comment:
Perhaps the correct fix would be using `glue_file.string()` but that may require updating Boost, as they has been some change in Boost filesystem since this code was written and in this area.